docs/src/pages/components/button/demo.tsx

```diff
@@ -1,114 +1,166 @@
-import React from 'react';
-import { Button, ButtonSizes, ButtonVariations } from '@aws-amplify/ui-react';
-import { FieldLabeler } from '@/components/FieldLabeler';
-import { Example } from '@/components/Example';
+import * as React from 'react';
+
+import {
+  Button,
+  ButtonSizes,
+  ButtonVariations,
+  ButtonProps,
+  Flex,
+  SwitchField,
+  SelectField,
+  TextField,
+} from '@aws-amplify/ui-react';
+
+import { Demo } from '@/components/Demo';
+
+const propsToCode = (props) => {
+  return (
+    `<Button` +
+    (props.disabled ? `\n  isDisabled={${props.disabled}}` : '') +
```

Comment:
Will we need to keep this in sync with [152-163](https://github.com/aws-amplify/amplify-ui/pull/759/files#diff-f85e0a9930f64b27b62d72172a82b9d915145c2ac8db04e94c178e0664e3cc8cR152-R163) below? Feels a bit awkward

Reply:
Yea I wonder if there is a better way to do this...